pitch.md

```diff
@@ -1,44 +1,90 @@
 
+We need a DSL for this because we want to enable users to specify their drinks to arbitrary levels of precision. There are so many factors that go into a caffeinated or alcoholic beverage that we need a language (rather than, for instance, an enumerated type), because there are just so many possibilities. I imagine that there could be a GUI for end users to interact with this system, but compiling to a DSL built specifically for this purpose seems advantageous. For this project in particular, I plan not to build a GUI (because that seems like more trouble than it's worth), but you could certainly imagine building a GUI on top of the basic language.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I **love** coffee, so I'd be really excited to actually make this kind of thing. Amusingly (and probably not coincidentally), I came up with it while sitting in a coffee shop. An important project consideration for me is building something that I think could ulimately be useful, and I think this language could fill a currently empty niche in the drinks market. A barista that I discussed the idea with suggested making the language apply to alcoholic drinks as well, because they're (in his opinion) easier to make algorithmically.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain? Be careful
 to distinguish between, e.g,. a graphical interface and a linguistic interface._ 
 
+I think the language is almost definitely going to have to be an external DSL - it doesn't really make sense to me to have it as an internal DSL, since its syntax is so different from Scala's (the language I plan to build my project in). In addition, after completing the assignments piconot and piconot-external, I have a feeling that it might be easier for me to implement this language as an external DSL - it felt like Scala's features hindered more than helped when we were implementing Picobot as an internal DSL, and I see no reason why something different would happen for this language. 
+
+For CS111, I'm going to make the language interactable only through a text-based program. Programming would involve typing commands in the syntax of the language. I think this is the right way to create the language as a project for CS111 because creating a GUI (though it may be more user friendly) would be adding an additional level of complication that doesn't have much to do with language design, which is meant to be the focus of this project. If you wanted to make this language more usable for the general public, though, it would probably be advantageous to build a GUI for it eventually.
+
+A program might look something like this:
+```
+Begin LATTE:
+1. 2 shots espresso
+2. Add 10 oz 2% milk
+3. Add 2 oz foam
+4. Add 4 pumps sugar
+End LATTE:
+```
+
+Or like this:
+
+```
+Begin PUMPKIN_SPICE_LATTE:
+1. LATTE
+2. Add 2 pumps pumpkin
```

Comment:
I talked to Prof Ben about this, too, and I think weights may be the better way to go initially. One concern I had about weights is that people may not know the weights of some of the ingredients of the drink (particularly the flavorings). For instance, what proportion of a 16oz drink is 2 pumps of pumpkin, really? 
This would be good motivation for eventually adding support for weights **or** exact measurements, even if I won't be able to do that on the first pass.